console/console-init/ui/src/modules/project-detail/components/ProjectCertificateList/ProjectCertificateToolbar.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  DataToolbar,
+  DataToolbarContent,
+  DataToolbarItem,
+  Button,
+  Switch
+} from "@patternfly/react-core";
+
+export interface IProjectCertificateToolbarProps {
+  setShowCertificateForm: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
is there any specific reason for using type React.Dispatch<React.SetStateAction<boolean>> ?

Reply:
I am passing the setter function returned by `useSate` as a prop.